framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -56,6 +57,26 @@
     /** The empty range. */
     public static final Range NOTHING = new Range();
 
+    /**
+     * Convenience construtor for often invoked pattern for Long collections
+     *
+     * @param collection the min and max value found in it will be used as from and to values in the
```

Comment:
Please say what the argument is.  This mentions some facts about elements in the collection but doesn't directly document the collection.  (The summary documentation on line 61 should do so too.)